week-6/Homework/mandatory/2-quotegenerator/quotes.js

```diff
@@ -1,3 +1,44 @@
+const newQuoteBtn = document.querySelector("#new-quote-btn");
+newQuoteBtn.addEventListener("click", displayRandomQuote);
+
+function displayRandomQuote() {
+  const randomQuoteObj = pickFromArray(quotes);
+
+  const quoteHolder = document.querySelector("#quote");
+  quoteHolder.textContent = randomQuoteObj.quote;
+
+  const quoteAuthorHolder = document.querySelector("#author");
+  quoteAuthorHolder.textContent = "- " + randomQuoteObj.author;
+}
+
+let autoPlayOn = false;
+let autoPlayInterval = -1;
```

Comment:
Note that `autoPlayInterval` is already assigned an actual interval ID before is ever read so it's fine to not assign anything to it here